docs/core/diagnostics/runtime-exception-events.md

```diff
@@ -0,0 +1,152 @@
+# Exception Events
+These runtime events capture information about exceptions that are thrown.
+
+This category consists of the following events:
+
+- [ExceptionThrown_V1 Event](#exceptionthrown_v1-event)
+- [ExceptionCatchStart Event](#exceptioncatchstart-event)
+- [ExceptionCatchStop Event](#exceptioncatchstop-event)
+- [ExceptionFinallyStart Event](#exceptionfinallystart-event)
+- [ExceptionFinallyStop Event](#exceptionfinallystop-event)
+- [ExceptionFilterStart Event](#exceptionfilterstart-event)
+- [ExceptionFilterStop Event](#exceptionfilterstop-event)
+- [ExceptionThrownStop Event](#exceptionthrownstop-event)
+
+## ExceptionThrown_V1 Event
+|Keyword for raising the event|Level|  
+|-----------------------------------|-----------|  
+|`ExceptionKeyword` (0x8000)|Error (1)|  
+  
+ The following table shows event information.  
+  
+|Event|Event ID|Raised when|  
+|-----------|--------------|-----------------|  
+|`ExceptionThrown_V1`|80|A managed exception is thrown.|  
+
+|Field name|Data type|Description|  
+|----------------|---------------|-----------------|  
+|Exception Type|win:UnicodeString|Type of the exception; for example, `System.NullReferenceException`.|  
+|Exception Message|win:UnicodeString|Actual exception message.|  
+|EIPCodeThrow|win:Pointer|Instruction pointer where exception occurred.|  
+|ExceptionHR|win:UInt32|Exception [HRESULT](https://docs.microsoft.com/openspecs/windows_protocols/ms-erref/0642cb2f-2075-4469-918c-4441e69c548a).|  
+|ExceptionFlags|win:UInt16|`0x01`: HasInnerException.<br /><br /> `0x02`: IsNestedException.<br /><br /> `0x04`: IsRethrownException.<br /><br /> `0x08`: IsCorruptedStateException (indicates that the process state is corrupt; see [Handling Corrupted State Exceptions](https://docs.microsoft.com/archive/msdn-magazine/2009/february/clr-inside-out-handling-corrupted-state-exceptions)).<br /><br /> `0x10`: IsCLSCompliant (an exception that derives from <xref:System.Exception> is CLS-compliant; otherwise, it is not CLS-compliant).|  
```

Comment:
```suggestion
|ExceptionFlags|win:UInt16|`0x01`: HasInnerException.<br /><br /> `0x02`: IsNestedException.<br /><br /> `0x04`: IsRethrownException.<br /><br /> `0x08`: IsCorruptedStateException (indicates that the process state is corrupt; see [Handling Corrupted State Exceptions](/archive/msdn-magazine/2009/february/clr-inside-out-handling-corrupted-state-exceptions)).<br /><br /> `0x10`: IsCLSCompliant (an exception that derives from <xref:System.Exception> is CLS-compliant; otherwise, it is not CLS-compliant).|  
```
We are removing the docs.microsoft.com from links so that they work in air-gapped environments too.